code_generation/templates/power_grid_model_c/power_grid_model/include/power_grid_model/auxiliary/meta_gen/attribute_classes.hpp.jinja

```diff
@@ -13,12 +13,13 @@
 #include "../../power_grid_model.hpp" // NOLINT
 #include "../../three_phase_tensor.hpp" // NOLINT
 #include "../meta_data.hpp" // NOLINT
+#include "gen_getters.hpp" // NOLINT
```

Comment:
If I try to format the generated code. It format exactly the other way around as opposed to what you mentioned, see below. Is this correct?

```c++
#include "../../enum.hpp"               // NOLINT
#include "../../power_grid_model.hpp"   // NOLINT
#include "../../three_phase_tensor.hpp" // NOLINT
#include "../input.hpp"                 // NOLINT
#include "../meta_data.hpp"             // NOLINT
#include "gen_getters.hpp"              // NOLINT
```

Reply:
what we usually do is add an additional whiteline.

The underlying thing is that the second you put something in another directory, you kind of create a structure that's in some way similar to creating a different interface library. In that regard, anything in that parent directory can be treated as being in a separate library. Because of the same reason, I personally tend to use a separate namespace for every subdirectory.

See e.g. https://github.com/PowerGridModel/power-grid-model/blob/main/power_grid_model_c/power_grid_model/include/power_grid_model/component/branch.hpp#L9-L17

Reply:
done